redis/tests/test_basic.rs

```diff
@@ -1494,6 +1494,21 @@ mod basic {
         assert_args!(&opts, "EX", "1000");
     }
 
+    #[test]
```

Comment:
I think the test would be stronger if you used `SET` w/ `EXAT`/`PXAT` and then check to make sure the times match with the new commands.

Reply:
@jaymell Thanks for your review, will change it. And the CI failure is that Redis 7.2 doesn't support those commands yet, will check the version before running tests.